.github/workflows/continuous-integration.yml

```diff
@@ -78,21 +78,22 @@ jobs:
         run: ./scripts/test
 
   docs:
+    if: matrix.python-version != "3.11"
```

Comment:
the hvplot notebook fails on 3.11 I think there is some dependency issue that they need to fix upstream :(

Reply:
Can you add a comment to that effect to remind future us of why we need this `if`?